editor/export/project_export.cpp

```diff
@@ -1385,6 +1543,31 @@ ProjectExportDialog::ProjectExportDialog() {
 			exclude_filters);
 	exclude_filters->connect(SceneStringName(text_changed), callable_mp(this, &ProjectExportDialog::_filter_changed));
 
+	// Patch packages.
+
+	VBoxContainer *patch_vb = memnew(VBoxContainer);
+	sections->add_child(patch_vb);
+	patch_vb->set_name(TTR("Patches"));
+
+	patches = memnew(Tree);
+	patch_vb->add_child(patches);
+	patches->set_v_size_flags(Control::SIZE_EXPAND_FILL);
+	patches->set_hide_root(true);
```

Comment:
```suggestion
	patches->set_v_size_flags(Control::SIZE_EXPAND_FILL);
	patches->set_hide_root(true);
	patches->set_auto_translate_mode(AUTO_TRANSLATE_MODE_DISABLED);
```

This `Tree` is for filenames, which should not be translated automatically.